### DISCUSSION TITLE
maybe android platform should be considered.

### DISCUSSION THREAD

**Discussion by colornix** [JanSeliv/Bomber#68] [Ideas]
since I donot like play bomberman on computer,  I suggest move the game to android

**Comment by JanSeliv** [comment]
Thanks for bringing up the Android support topic!

I want to make it clear right off the bat: you can actually package the game for Android right now, and it runs fine on Android devices without crashing during gameplay.

See the video of current state:
https://github.com/JanSeliv/Bomber/assets/20540872/54561b2a-1291-4f14-a21e-ed632f5b6672

From the video you can notice there are few critical issues:

- **Performance/lighting**: The game doesn't run as smoothly on Android as it could. Making games work well on Android usually means doing a lot of optimizations, especially with lightning that is broken right now in shown Android build. We would need to do the pass on materials, LODs, scalability levels etc. Huge amount of work.

- **Controls**: The UI buttons in the game's menus are working fine, but virtual thumbsticks are not implemented, so player can not move and put bombs at this moment in Android build. It could be implemented relatively easily.

Having these issues, I'm not sharing a ready-to-use .APK file, as I'm unable to dedicate additional time to enhance the Android version at the moment. Anytime soon I'm personally focused more on finishing core gameplay itself.

I would appreciate any assistance from the community on improving Android experience since the game is already up and running, offering a good foundation.
